src/entry.py

```diff
@@ -5,6 +5,7 @@
 
 import pandas as pd
 from rich.table import Table
+from rich_tools import table_to_df
```

Comment:
awesome find!

Reply:
Oh actually I read it as rich.tools, is this a new package getting installed?
Need to update requirements.txt in that case

Reply:
yeah it is a new package, will update requirements.txt